README.md

```diff
@@ -48,7 +48,7 @@ Here's to a thrilling Hacktoberfest voyage with us! 🎉
 # Decentralized Web Node (DWN) SDK <!-- omit in toc -->
 
 Code Coverage
-![Statements](https://img.shields.io/badge/statements-98.48%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-95.56%25-brightgreen.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-95.71%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-98.48%25-brightgreen.svg?style=flat)
+![Statements](https://img.shields.io/badge/statements-98.5%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-95.82%25-brightgreen.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-95.76%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-98.5%25-brightgreen.svg?style=flat)
```

Comment:
Sanity: where did the `codecov` badge go?

We should just use that one going forward.

Reply:
Updated -- I accidentally switched it back to the old badges in another PR